reference/release-notes/1.23.0.md

```diff
@@ -0,0 +1,76 @@
+
+## Removed functionality
+
+< List features that were part of Anbox Cloud so far or that were on the deprecation path for a while and have been removed for the current release >
+
+## Deprecations
+
+* Use of a node controller to manage port forwarding from an instance is deprecated, starting with this release. In 1.23.0, the node controller will still be used for the full Anbox Cloud deployment. For the appliance, the Anbox Management Service (AMS) now uses a LXD proxy device to manage the port forwarding from an instance to expose a service externally or internally.
+
+  In the 1.24.0 release, the node controller will be fully replaced with LXD proxy device for both regular and appliance deployments.<!--AC--2487-->
+
+## Known issues
+
+* WiFi support is not available with Android 14 vendor images. This issue will be fixed in the 1.23.1 release. <!--AC-2647-->
+
+## Bug fixes
+- [LP 2058004](https://bugs.launchpad.net/anbox-cloud/+bug/2058004) Deploying Anbox Cloud dashboard fails when using AMI that has a Ubuntu Pro subscription.
+- [LP 2065281](https://bugs.launchpad.net/anbox-cloud/+bug/2065281) Android 13 unable to use Vulkan 1.3 despite Vulkan 1.3 being supported.
+- [LP 2067314](https://bugs.launchpad.net/anbox-cloud/+bug/2067314) Current `amc shell` is a plain bash shell and will not source any common profile scripts.
+- [LP 2067316](https://bugs.launchpad.net/anbox-cloud/+bug/2067316) The `pprof` endpoint exposed by AMS for profiling heap allocations does not expose `allocs` handler.
+- [LP 2072955](https://bugs.launchpad.net/anbox-cloud/+bug/2072955) Unable to add an image on Anbox Cloud Appliance.
+- [LP 2067252](https://bugs.launchpad.net/anbox-cloud/+bug/2067252) Incorrect  `Error: api extension "zip_archive_support" not supported` error while creating an application.
+- [LP 2069021](https://bugs.launchpad.net/anbox-cloud/+bug/2069021) Updating the AMS SDK to 1.21.0 prevents building the AMS client.
+- [LP 2070252](https://bugs.launchpad.net/anbox-cloud/+bug/2070252) When running the Marvel Battle application, the following logs flooded the Anbox logcat:
+    ```
+    06-13 12:12:16.403 3221 3221 D SensorMonitor: Temperature Sensor value changed
+    06-13 12:12:16.408 3221 3221 D SensorMonitor: Temperature Sensor value changed
+    06-13 12:12:16.413 3221 3221 D SensorMonitor: Temperature Sensor value  changed
+    ```
+- [LP 2072367](https://bugs.launchpad.net/anbox-cloud/+bug/2072367) When initialising the Anbox Cloud Appliance, if you provide an empty public address, invalid STUN/TURN server details are generated.
+- [LP 2075494](https://bugs.launchpad.net/anbox-cloud/+bug/2075494) The streaming screen does not rotate when using client side upscaling.
```

Comment:
This is still in triaged state in LP, is this included in 1.23.0? @morphis @jat-canonical

Reply:
No, it's not. The priority for this bug is pretty low also Samsung now has a way to work around this anyway.